### PR TITLE
Pre-load each graph image before updating the dashboard UI

### PR DESCRIPTION
**Pull request by davidhusselmann**
This stops the Dashboard screen from flashing blank images and being distracting to look at.

### CODE REVIEW

**Comment by deniszh** [comment]
I'm 🤷‍♂️ about that change. Any opinions? /cc @graphite-project/committers ?

**Comment by davidhusselmann** [comment]
I just figured I'd make a quick change to make it easier for me to use (my graphite server is quite slow so the images take a few seconds to load up) - figured I'd PR it in case the community can also use it.

**Comment by DanCech** [comment]
seems reasonable enough at first glance

**Comment by deniszh** [comment]
OK, will merge tomorrow if looks good

File: webapp/content/js/dashboard.js [hunk 1]
```diff
@@ -1070,7 +1070,14 @@ function updateGraphRecords() {
     if (!params.uniq === undefined) {
         delete params["uniq"];
     }
-    item.set('url', document.body.dataset.baseUrl + 'render?' + Ext.urlEncode(params));
+	
+	//Preload the image and set it into the UI once it is available.
+	var img = new Image();
```
Comment (iksaif) [inline webapp/content/js/dashboard.js]: indentation seems off here

**Comment by deniszh** [comment]
Ok, merging that. Ident is not so critical in JS, we can fix it later.

**Comment by davidhusselmann** [comment]
Thanks @deniszh.  I might do another enhancement to this soon as it isn't too obvious when images have finished loading when you do a full screen reload (not automatic refresh).

**Pull request merged**